src/Microsoft.DotNet.Darc/DarcLib/VirtualMonoRepo/VmrCloneManager.cs

```diff
@@ -51,24 +53,25 @@ public VmrCloneManager(
     }
 
     public async Task<ILocalGitRepo> PrepareVmrAsync(
-        IReadOnlyCollection<string> remoteUris,
+        string remoteUri,
```

Comment:
Why did this change? I think we will want to clone the VMR with multiple remotes eventually

Reply:
I think that was only needed when we were able to have only one VMR on the service. With the change we did a bit ago, where we now have multiple VMRs on disk, I think it might not be needed anymore?
But I can revert the change, it's not a part of this PR anyway

Reply:
reverted